### ISSUE TITLE
linkblog.json not updating

### ISSUE THREAD

**Issue by trickyskills** [scripting/Scripting-News#238]
Hey there

I'm having a few issues with my linkblog.json updating. When I add a new entry in radio 3, I hit the Build Linkblog JSON from the menu. Nothing happens. New feeds are appearing on my radio 3 blog page, but not in the JSON file.

http://radio3.io/users/Jason_Cobb/

http://radio3.io/users/Jason_Cobb/linkblog.json

I've tried this in Chrome, Safari and Firefox, but still no joy.

Any ideas, anyone please?

Thanks

**Comment by scripting** [comment]
I just tried it myself and it worked.

http://twitter.radio3.io/users/davewiner/linkblog.json

Do you see any errors in the JavaScript console?

I assume you're logged in?

**Comment by trickyskills** [comment]
Thanks for the reply.

Yes - a couple of errors in the console.

https://wtcblog.co.uk/images/drummer/170922/error1.png

https://wtcblog.co.uk/images/drummer/170922/error2.png

The dates on these look really old? I use to gave an old fargo account...

I'm logged into radio 3 via Twitter as well.

Thanks

**Comment by scripting** [comment]
I don't have a lot of time to look into it, but have you checked if you have any antivirus software installed that might be crippling your connection?

**Comment by trickyskills** [comment]
No, nothing antivirus wise that is getting in the way. I think it is to do with having an old fargo account linked to the same Twitter handle. No worries, Dave. I can live without the Links. Thanks for your help and the software.

**Comment by scripting** [comment]
I can't imagine what a Fargo account would have to do with Radio3. I doubt if that's it. ;-)

**Comment by trickyskills** [comment]
It seems to be working now! No idea how or why. It's sometimes best not to ponder. Thanks again.

**Comment by scripting** [comment]
Hah! That's the way it always works. ;-)
